docker-compose.yml

```diff
@@ -382,6 +383,14 @@ services:
         condition: service_healthy
       provisioner:
         condition: service_completed_successfully
+      pulumi:
+        # We must wait until Pulumi has created the buckets to know what their names are
+        condition: service_completed_successfully
+    volumes:
+      - type: volume
+        source: pulumi_outputs
+        target: /pulumi-outputs
```

Comment:
(Not a must-do, but I sort of would prefer if target were `/mnt/pulumi-outputs` so it's more obvious this is a volume). However, I recognize I'm asking to reinvent the existing wheel - more something to consider in the future.

Reply:
I could take another pass through everything and do that, on a separate PR.